prime-router/docs/design/proposals/0023-condition-to-code-mapping/0023-condition-to-code-mapping.md

```diff
@@ -0,0 +1,383 @@
+# Proposal for Code-to-Condition Mapping
+
+## Background
+
+Public Health reporting in the United States is not uniform in nature with each public health agency (PHA) setting their
+own criteria for what conditions need to be reported to that jurisdiction. In the context of electronic lab reporting (ELR), the onus is typically on the reporting entity (lab, hospital, clinic etc.) to determine which result messages qualify for each PHA the entity reports to.
+
+Since ReportStream takes on the burden of identifying the appropriate destination for sender’s messages, it will be necessary to implement a feature to ensure only messages for the appropriate conditions/diseases are allowed to route to each connected receiver. This feature will be similar to the COVID-19 LIVD table in that it will be a table of LOINC codes and their associated conditions that is used to reference received LOINC/SNOMED codes and return their associated condition.
+
+## Assumptions
+1.) The Council of State and Territorial Epidemiologist (CSTE) ValueSets that make up the Reportable Conditions Trigger Codes (RCTC) will be sufficient to capture a majority of codes sent to ReportStream. If this turns out to be false, we can modify the design to include additional data sources.
+
+## Criteria
+
+1.) Must be able to map LOINC and SNOMED codes to condition.<br>
+2.) Tables must have the ability to be updated without a PR.<br>
+3.) Mapping must account for all 160 conditions available in Report Content Knowledge Management System that relate to ELR (https://www.rckms.org/conditions-available-in-rckms/) <br>
+4.) Must be able to add ad-hoc mappings as needed for local codes/LDTs or other non-standard codes
+
+## Out of Scope
+The below items are not covered in this proposal.
+
+1.) Mechanism to automate which conditions each state considers reportable.
+
+## Design
+
+Overview Diagram
+
+![img.png](code-to-condition diagram.png)
```

Comment:
Probably should include the "pruning" feature in diagram? Also check for spelling mistakes.